src/App.css

```diff
@@ -47,11 +58,184 @@ tr {
   padding-top: 15px;
 }
 
+.card {
+  width: 20rem;
+  border: 3px solid rgb(158, 141, 92);
+ 
+}
+ .infocards{
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  column-gap: 50px;
+  row-gap: 50px;
+  margin: 3rem;
+ }
+
+ .info-image{
```

Comment:
Try to leave a space between the  classname and the bracket if you can 😄

Reply:
Gotcha. x